servicetalk-examples/http/helloworld/src/main/java/io/servicetalk/examples/http/helloworld/async/HelloWorldClient.java

```diff
@@ -16,21 +16,34 @@
 package io.servicetalk.examples.http.helloworld.async;
 
 import io.servicetalk.http.api.HttpClient;
+import io.servicetalk.http.api.SingleAddressHttpClientBuilder;
 import io.servicetalk.http.netty.HttpClients;
+import io.servicetalk.http.utils.TimeoutHttpRequesterFilter;
+import io.servicetalk.transport.api.HostAndPort;
 
+import java.net.InetSocketAddress;
+import java.time.Duration;
 import java.util.concurrent.CountDownLatch;
 
 import static io.servicetalk.http.api.HttpSerializationProviders.textDeserializer;
+import static io.servicetalk.http.utils.TimeoutHttpRequesterFilter.useDefaultTimeout;
 
 public final class HelloWorldClient {
 
     public static void main(String[] args) throws Exception {
-        try (HttpClient client = HttpClients.forSingleAddress("localhost", 8080).build()) {
+        SingleAddressHttpClientBuilder<HostAndPort, InetSocketAddress> builder =
+                HttpClients.forSingleAddress("localhost", 8080)
+                // (optional) All requests must fully complete within 2 minutes
```

Comment:
lets keep the hello world example simple, and add a new example if we want to demonstrate new features.

Reply:
OK, easily done.